### PR TITLE
 add conda based jupyterlab Dockerfile

### PR DESCRIPTION
**Pull request by mklkun**

### CODE REVIEW

File: jupyterlab/Dockerfile.conda [hunk 1]
```diff
@@ -0,0 +1,33 @@
+FROM continuumio/anaconda3
+
+WORKDIR /root
+
+RUN apt-get update && apt-get install -y
```
Comment (mcompastie) [inline jupyterlab/Dockerfile.conda]: Why apt-get install -y ?

File: jupyterlab/Dockerfile.conda [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+# Install java
+# ------------
+RUN apt-get update && apt-get install -y openjdk-8-jdk
```
Comment (mcompastie) [inline jupyterlab/Dockerfile.conda]: Delete any redundancy

File: jupyterlab/Dockerfile.conda [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+# Install ProActive Jupyter Kernel
+# ------------------------
+RUN pip install proactive proactive-jupyter-kernel --upgrade
+RUN python -m proactive-jupyter-kernel.install
+
+ADD jupyter_notebook_config.py /root/.jupyter/jupyter_notebook_config.py
+
+EXPOSE 8888
+
+ENTRYPOINT ["jupyter", "lab", "--port=8888", "--ip=0.0.0.0", "--no-browser", "--allow-root", "--NotebookApp.token=''", "--NotebookApp.password=''"]
```
Comment (mcompastie) [inline jupyterlab/Dockerfile.conda]: Optional: wouldn't be worthy to configure token and password as environment configuration ?

File: jupyterlab/README.md [hunk 4]
```diff
@@ -1,14 +1,33 @@
...
+
+```aidl
+docker build -t activeeon/jupyterlab:pip -f ./Dockerfile.pip .
 ```
-docker build -t activeeon/jupyterlab .
+
+#### Docker run
+
+```aidl
+docker run -it -p 8888:8888 activeeon/jupyterlab:pip
```
Comment (mboussaa) [inline jupyterlab/README.md]: add --name

File: jupyterlab/README.md [hunk 5]
```diff
@@ -1,14 +1,33 @@
...
+
+```aidl
+docker build -t activeeon/jupyterlab:conda -f ./Dockerfile.conda .
 ```
-docker run -it -p 8888:8888 activeeon/jupyterlab
+
+#### Docker run
+
+```aidl
+docker run -it -p 8888:8888 activeeon/jupyterlab:conda
```
Comment (mboussaa) [inline jupyterlab/README.md]: do you really need interactive mode? detached maybe?

File: jupyterlab/Dockerfile.conda (hunk 3 above)
Comment (andrewssobral) [reply jupyterlab/Dockerfile.conda]: @mcompastie this docker will be used as a open jupyter service, and by default we need to desactivate the login and password.

File: jupyterlab/README.md (hunk 5 above)
Comment (andrewssobral) [reply jupyterlab/README.md]: @mboussaa yes...this docker container is intended to be used as a open service.

File: jupyterlab/README.md (hunk 4 above)
Comment (andrewssobral) [reply jupyterlab/README.md]: @mboussaa I think it's not necessary into this point...

File: jupyterlab/Dockerfile.conda [hunk 6]
```diff
@@ -0,0 +1,33 @@
+FROM continuumio/anaconda3
+
+WORKDIR /root
+
+RUN apt-get update
+
+# Install java
+# ------------
+RUN apt-get install -y openjdk-8-jdk
+RUN apt-get install ca-certificates-java && update-ca-certificates -f
```
Comment (mcompastie) [inline jupyterlab/Dockerfile.conda]: Merge L9 + L10, ensure apt is launched in non interactive mode.

File: jupyterlab/Dockerfile.conda [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+RUN apt-get clean
+
+# Install ProActive library
+# ------------------------
+RUN pip install -U proactive --upgrade
+
+# Install Graphviz
+# ------------------------
+RUN apt-get install -y graphviz-dev
+RUN conda install graphviz
```
Comment (mcompastie) [inline jupyterlab/Dockerfile.conda]: Is conda run in non interactive mode ?

File: jupyterlab/Dockerfile.conda (hunk 6 above)
Comment (mklkun) [reply jupyterlab/Dockerfile.conda]: The merge of L9 and L10 gives some unexpected errors.

**Pull request merged**